test/v2/client/test_client.go

```diff
@@ -14,7 +14,9 @@ import (
 	"github.com/Layr-Labs/eigenda/api/clients/v2/payloaddispersal"
 	"github.com/Layr-Labs/eigenda/api/clients/v2/payloadretrieval"
 	relayv2 "github.com/Layr-Labs/eigenda/api/clients/v2/relay"
```

Comment:
nit: suggest keeping default name `relay`, instead of manually renaming to `relayv2`

Reply:
fixed